### ISSUE TITLE
HowTo generate Pair of API Keys?

### ISSUE THREAD

**Issue by maxicode18** [zircote/swagger-php#593]
Hi,
I plan to implement AppId and ApiKey auth in my symfony project.
At the moment I have only API-Key auth.

Now I need APP-ID and ApiKey auth.

How to note this in the swagger-php doc block?

The Swagger specification is this:
```
securityDefinitions:  
  apiKey:  
    type: apiKey  
    in: header  
    name: X-API-KEY  
  appId:  
    type: apiKey  
    in: header  
    name: X-APP-ID
```
https://swagger.io/docs/specification/authentication/api-keys/

I use Swagger PHP 3.0
Thank you.

**Comment by bfanger** [comment]
As described in **Multiple API Keys** section.

```php
/**
 * @OA\OpenApi(
 *   security={{"apiKey":{},"appId":{}}}
 * )
 */
```

Currently swagger-php serializes that as empty objects instead of empty arrays.
Help with fixing that issue is appreciated.

**Comment by bfanger** [comment]
Fixed in 3.0.2
